avr/cores/megacommand/MD/MD.cpp

```diff
@@ -187,21 +190,25 @@ bool MDClass::get_fw_caps() {
 void MDClass::activate_trig_interface() {
   uint8_t data[3] = {0x70, 0x31, 0x01};
   sendRequest(data, sizeof(data));
+  waitBlocking();
```

Comment:
These are asynchronous requests. why do we need to wait ?

Reply:
```
send gateway command
send getBlocking command
waiting for callback to trigger
gateway command replies
callback accepted the sysex command
read the wrong message
getBlocking command replies, and ignored
```

Reply:
I don't follow,

activate_trig_interface() just sends a sysex message to the MD.

There should be no waiting after this ?

Reply:
> activate_trig_interface() just sends a sysex message to the MD.

Each 0x72 gateway message has a reply.

Reply:
ah i see that now.

Reply:
> The two messages dumped from end_immediate (isr):
> 
> ```
> getByte(5) = 114 (0x72)
> getByte(6) = 34 (0x22)
> getByte(7) = 2 (0x2)
> getByte(8) = 0 (0x0)
> 
> getByte(5) = 114 (0x72)
> getByte(6) = 30 (0x1E)
> getByte(7) = 0 (0x0)
> getByte(8) = 0 (0x0)
> ```

Is this undefined behaviour? Looking at the MD side we have a switch statement, but there's no default case for when the message type is unrecognized.